core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.html

```diff
@@ -78,4 +78,31 @@
 # once a day on the 1st and 15th of every month except December
 H H 1,15 1-11 *
 </pre>
+<p>
+  <b>Timezones specification</b>. <br/>
+  Starting from 1.615 Jenkins supports the specification 
+  of different timezones in this field. It allows to natively configure all
+  periodic operations.
+  
+  <ul>
+    <li>Timezone can be specified in the first line, which should contain the &quot;TZ=$(timezoneID)&quot; string</li>
```

Comment:
I would be a bit more explicit that it *must* be the first line, and maybe we should avoid hinting at the `$(...)` syntax, as I don't believe this field can resolve environment variables.

Something like this, perhaps?
```
<li>The timezone must be specified on the first line of this field, in the format: <tt>TZ=&lt;timezone ID&gt;</tt></li>
<li>All subsequent scheduled times will be interpreted using this timezone</li>
```